.ruby-version

```diff
@@ -1 +1 @@
-2.1
+2.4
```

Comment:
I think we should downgrade this to 2.3 at this point.

Reply:
In truth, it seems that this file doesn't affect anything in the build. I would prefer remove it at this point.